hadoop-ozone/interface-client/src/main/proto/OmClientProtocol.proto

```diff
@@ -1072,6 +1074,16 @@ message KeyInfo {
     optional bool isFile = 19;
 }
 
+message BasicKeyInfo {
+    required string keyName = 1;
+    required uint64 dataSize = 2;
+    required uint64 creationTime = 3;
+    required uint64 modificationTime = 4;
+    required hadoop.hdds.ReplicationType type = 5;
+    optional hadoop.hdds.ReplicationFactor factor = 6;
```

Comment:
This can be populated server side only if it differs from the bucket's config. This should reduce the payload and processing server-side unless the key is replicated in config different from the bucket's default.

Reply:
Understood, thank you for the review. I am working on making the required changes!

Reply:
I was planning on addressing this comment as part of a follow-up PR since there are several file changes required for the same.

I have a forked branch: [HDDS-9079_followup](https://github.com/tanvipenumudy/ozone/commits/HDDS-9079_followup) containing the necessary changes, thanks!

Reply:
I think we can do this as an optimization post this PR. The biggest bang is already part of this PR, and we should complete the end-to-end get this merged, and measure any incremental performance improvements. We would need to do a get bucket info which might negate any performance gains.